profiles/common/modules/features/multisite_forum_community/multisite_forum_community.features.inc

```diff
@@ -9,9 +9,13 @@
  * Implements hook_ctools_plugin_api().
  */
 function multisite_forum_community_ctools_plugin_api($module = NULL, $api = NULL) {
+
```

Comment:
what's the empty space for ?

Reply:
Unrelated changes.

Reply:
I've been fighting with this since 3 years...